packages/uicore/src/components/TableV2/TableV2.tsx

```diff
@@ -58,7 +60,9 @@ export const TableV2 = <Data extends Record<string, any>>(props: TableProps<Data
     rowDataTestID,
     getRowClassName,
     name,
-    renderRowSubComponent
+    renderRowSubComponent,
+    autoResetSortBy = true,
```

Comment:
Are they true by default? If not then this might be a breaking change

Reply:
Yes, they are both true by default ([doc1](https://react-table-v7.tanstack.com/docs/api/useSortBy#table-options), [doc2](https://react-table-v7.tanstack.com/docs/api/usePagination#table-options)).